package.json

```diff
@@ -14,19 +24,12 @@
     "test": "TZ=UTC jest",
     "postinstall": "husky install"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/amir20/dozzle.git"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/amir20/dozzle/issues"
-  },
-  "homepage": "https://github.com/amir20/dozzle#readme",
   "dependencies": {
     "@babel/core": "^7.15.8",
     "@babel/plugin-transform-runtime": "^7.15.8",
+    "@iconify-json/cil": "^1.0.1",
```

Comment:
I think you can remove these and configure `autoInstall: true` instead for icons. They will be automatically installed.

Reply:
You're right. Done. For some reason it doesn't automatically add octicon set to the lockfile, but the octicon files are there.